crates/sncast/src/lib.rs

```diff
@@ -384,16 +382,19 @@ pub async fn handle_wait_for_tx<T>(
         .await
         {
             Ok(_) => Ok(return_value),
-            Err(message) => Err(anyhow!(message)),
+            Err(message) => {
+                eprintln!("Transaction hash: {transaction_hash:#x}\n");
+                Err(anyhow!(message))
```

Comment:
Why we have added this print here?

Reply:
The transaction hash is supposed to be printed in case the command fails, so the user will at least have the info about the transaction

Reply:
Printing it the previous way will result in `Transaction hash` being printed always and without the appropriate formatting (json)